missioncontrol/tests/test_files.py

```diff
@@ -204,4 +206,14 @@ def test_get_post_data_fields(boto3_mock, test_client, some_hash):
     boto3_mock.client.return_value.generate_presigned_post.assert_called_with(
         Bucket=settings.FILE_STORAGE_PATH.split('/')[2],
         Key=f'django-file-storage/{some_hash}',
-    )
\ No newline at end of file
+    )
+
+
+@pytest.mark.django_db
+def test_get_post_data_fields_local_not_implemented(settings, test_client, some_hash):
+    settings.FILE_STORAGE_PATH = '/tmp/'
```

Comment:
That's a very handy fixture, I was about to comment that that wouldn't get reverted.